src/sandbox.rkt

```diff
@@ -195,7 +197,11 @@
     (dummy-table-row result "timeout" link)]))
 
 (define (unparse-result row)
-  `(FPCore ,(table-row-vars row)
+  (define top
+    (if (table-row-identifier row)
+        (list (table-row-identifier row) (table-row-vars row))
+        (list (table-row-vars row))))
```

Comment:
Maybe I'm dense but why keep the `table-row-vars` in both branches?

Reply:
An FPCore either starts with `(FPCore (arg1 arg2 ...) ...)` or `(FPCore ident (arg1 arg2 ...) ...)`. Both require variables.